### PR TITLE
Added Linux readme

### PR DESCRIPTION
**Pull request by AMZN-Olex**
Signed-off-by: Olex Lozitskiy <5432499+AMZN-Olex@users.noreply.github.com>

### CODE REVIEW

File: README_LINUX.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+# Configure.
+> cmake .. -G "Ninja Multi-Config" -DCMAKE_BUILD_TYPE=profile -DLY_3RDPARTY_PATH=~/ws/3rdParty -DCMAKE_C_COMPILER=clang-12 -DCMAKE_CXX_COMPILER=clang++-12 -DLY_PROJECTS="~/git/o3de-multiplayersample"
+
+# Build the Editor, game launcher and server launcher.
+> cmake --build . --config profile --target Editor
+> cmake --build . --config profile --target MultiplayerSample.GameLauncher
+> cmake --build . --config profile --target MultiplayerSample.ServerLauncher
+```
+
+#### Step 3b. (Optional) Build and Run Multiplayer Unittests
```
Comment (AMZN-Gene) [inline README_LINUX.md:71]: Unit Tests (2 words)

File: README_LINUX.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+```shell
+> cd ~/git/o3de-multiplayersample/build
+
+> cmake --build . --config profile --target Multiplayer.Tests
+# Run unittests and benchmarks
```
Comment (AMZN-Gene) [inline README_LINUX.md:77]: unit tests

File: README_LINUX.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+host
+LoadLevel Levels/SampleBase/SampleBase.spawnable
+```
+
+### Step 5. Verify Asset Processor 
+
+```shell
+> cd ~/git/o3de-multiplayersample/build
+> cmake --build . --config profile --target AssetProcessor
+# launch Asset Processor and verify MultiplayerSample assets are good
```
Comment (AMZN-Gene) [inline README_LINUX.md:102]: capital “Launch” to match other comments starting uppercase

**Review by AMZN-Gene — APPROVED** [review]

File: README_LINUX.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+> mkdir build
+> cd build
+
+# Configure.
+> cmake .. -G "Ninja Multi-Config" -DCMAKE_BUILD_TYPE=profile -DLY_3RDPARTY_PATH=~/ws/3rdParty -DCMAKE_C_COMPILER=clang-12 -DCMAKE_CXX_COMPILER=clang++-12 -DLY_PROJECTS="~/git/o3de-multiplayersample"
+
+# Build the Editor, game launcher and server launcher.
+> cmake --build . --config profile --target Editor
+> cmake --build . --config profile --target MultiplayerSample.GameLauncher
+> cmake --build . --config profile --target MultiplayerSample.ServerLauncher
```
Comment (spham-amzn) [inline README_LINUX.md:68]: All 3 targets can be set in one build command line
```suggestion
> cmake --build . --config profile --target Editor MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher
```

File: README_LINUX.md (hunk 1 above)
Comment (AMZN-Olex) [reply README_LINUX.md]: Corrected.

File: README_LINUX.md (hunk 2 above)
Comment (AMZN-Olex) [reply README_LINUX.md]: Corrected.

File: README_LINUX.md (hunk 3 above)
Comment (AMZN-Olex) [reply README_LINUX.md]: Corrected.

File: README_LINUX.md (hunk 4 above)
Comment (AMZN-Olex) [reply README_LINUX.md]: Corrected.

**Review by spham-amzn — APPROVED** [review]

**Pull request merged**